.github/workflows/update_libs_merge.yaml

```diff
@@ -0,0 +1,25 @@
+# reusable workflow for automatically approving PRs updating charm libraries
+name: Approve and Merge Updating Charm Libraries
+run-name: CI for approving and merging automated-update-charm-libs PRs
+
+on:
+  push:
+    branches:
+      - "automated-update-charm-libs"
+
+jobs:
+  autoapprove:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Approve PR
+        run: |
+          gh pr review --approve || true
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
+      - name: Merge PR
+        # If pr can't be merged because requirements aren't met automerge is enabled
+        run: |
+          gh pr merge --auto --body "Update charm libs" --subject "(Automated) Update Charm Libs" || true
```

Comment:
You might want to move this part in here https://github.com/canonical/identity-platform-login-ui-operator/blob/main/.github/workflows/update_libs.yaml basically enable automerge on pr creation